libsolidity/analysis/TypeChecker.cpp

```diff
@@ -861,6 +861,24 @@ bool TypeChecker::visit(InlineAssembly const& _inlineAssembly)
 						return false;
 					}
 				}
+				else if (auto const* fpType = dynamic_cast<FunctionTypePointer>(var->type()))
+				{
+					if (suffix != "selector" && suffix != "address")
+					{
+						m_errorReporter.typeError(9272_error, _identifier.debugData->location, "Variables of type function pointer only support \".selector\" and \".address\".");
```

Comment:
I think we never oficially refer to these as pointers.
```suggestion
						m_errorReporter.typeError(9272_error, _identifier.debugData->location, "Variables of function types only support \".selector\" and \".address\".");
```